src/mqtt.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from "@bristom/logger";
+import mqtt from "async-mqtt";
+
+const iot = () => {
+  const client = mqtt.connect("mqtt://test.mosquitto.org");
+
+  const initMqtt = async () => {
+    Logger.debug("Starting MQTT");
+    try {
+      await client.subscribe("cora-hacktoberfest2022");
+      await client.publish("cora-hacktoberfest2022", "Hello mqtt"); // this will be inside meter/esp32
+    } catch (e) {
+      Logger.debug(`Something wrong with MQTT start: ${e}`);
+    }
+  };
+
+  client.on("connect", initMqtt);
+
+  const handleMqttMessage = async (topic, message) => {
+    // message is Buffer
+    Logger.debug(message.toString());
+    await client.end();
```

Comment:
Does this client.end here kills the connection with the MQTT server? If so, we would be able to only receive the first message sent by the meter, it doesn't make a lot of sense for me

Reply:
Yes, it ends it. Despite being only a demo, it makes sense to decide the logic here, right now in the beginning. Should we close this connection on a node process.exit, for example?

Reply:
@arthurborgesdev Actually the connection shouldn't be closed at all, you should watch the topic all the time for new messages and everytime one comes save it at the DB or else we would have to restart the app (or the connection itself) everytimes a message comes

Reply:
Acknowledged! Thanks for the explanation!